File: src/gestorAplicacion/usuario/Mascota.java
package gestorAplicacion.usuario;
import java.util.*;
import gestorAplicacion.logistica.*;

public class Mascota extends CargaExtra{

    private static ArrayList<Mascota> mascotas = new ArrayList<Mascota>();

    //constructores
    public Mascota(int precio, String tamaño) {
        super(precio, tamaño);
        agregar();
    }

    @Override
    public void agregar() {
        mascotas.add(this);
        
    }

    //getter and setter
    public static ArrayList<Mascota> getMascotas() {
        return mascotas;
    }
    public static void setMascotas(ArrayList<Mascota> mascotas) {
        Mascota.mascotas = mascotas;
    }
}

File: Python/src/gestorAplicacion/usuario/Mascota.py
from logistica import *
class Mascota(CargaExtra):
    mascotas= []
    def __init__(self,precio,tamaño):
        super(precio, tamaño)
        Mascota.mascotas.append(self)
    
    def agregar(self):
        Mascota.mascotas.append(self)
    
    @classmethod
    def getEquipajes(cls):
        return cls.mascotas
    @classmethod
    def setEquipajes(cls,mascota):
        cls.mascotas = mascota